.github/CONTRIBUTING.md

```diff
@@ -1,79 +1,132 @@
 # Contribution Guidelines
-# <p><a href="https://elementor.com/?utm_source=github-repo&utm_medium=link&utm_campaign=contribution">![logo](https://user-images.githubusercontent.com/1778512/191041718-728d179e-07cb-4cb4-953a-6c294ee8c4db.png)</a></p>
 
-Before filing a bug report, a feature request or sending pull requests, be sure to read the contribution guidelines.
+# <a href="https://elementor.com/?utm_source=github-repo&utm_medium=link&utm_campaign=contribution">![logo](https://user-images.githubusercontent.com/1778512/191041718-728d179e-07cb-4cb4-953a-6c294ee8c4db.png)</a>
```

Comment:
Why use `#` ? double H1 headings...

Reply:
I copied the header from the original file. So it came with two H1s. I've fixed it and will send another commit.

Thanks for spotting this!